R/outdata.R

```diff
@@ -43,7 +43,7 @@
 #' @export
 #'
 #' @examples
-#' metalite:::outdata(
+#' outdata(
```

Comment:
@nanxstats, would you feel we need a global search of `:::` and remove them. 

We had examples for internal function that use `:::`. For example 

https://github.com/Merck/metalite/blob/237da7f6219c5dd8db889bed159b857309a260ce/R/meta_check.R#L31

Reply:
ah, maybe they are OK, as I saw `@noRd` in those internal functions. Maybe we just need to check if there is `:::` in the `man` folder. :)

Reply:
I just checked all Rd files and found `:::` does not appear.